pkg/helpers/slice_helper.go

```diff
@@ -0,0 +1,12 @@
+package helpers
+
+func CleanStringSlice(slice []string) []string {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [exported function CleanStringSlice should have comment or be unexported](https://app.codacy.com/manual/TurnsCoffeeIntoScripts/jira-api-resource/pullRequest?prid=4777494)